.circleci/config.yml

```diff
@@ -92,16 +103,28 @@ jobs:
           root: .
           paths:
             - prisma
-  run-tests:
+  run-tests-unit:
     executor: executor
+    parallelism: 3
```

Comment:
since now you run your tests on multiple machines you could try to run them on smaller machines. You are using Medium right now. Try to change it to small :)